site/content/contribute/getting-started/branching.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Mattermost tick-tock Branching Strategy"
+date: 2019-06-18T00:00:00-04:00
+subsection: Getting Started
+---
+
+Mattermost uses a tick-tock release strategy where every other release is a "quality release" that only has bug fixes and no new features. 
```

Comment:
Could add a reference to docs: https://docs.mattermost.com/process/release-faq.html#release-overview.